docker/nginx/nginx.conf

```diff
@@ -42,7 +42,7 @@ http {
         '"$http_user_agent" $upstream_response_time '
         '$upstream_bytes_sent $upstream_bytes_received '
         '"$upstream_http_content_type" "$upstream_cache_status" '
-        '"$portal_domain" "$upstream_http_skynet_skylink" '
+        '"$portal_domain" "$sent_http_skynet_skylink" '
```

Comment:
this is neat - didn't know that

Reply:
it would have saved me like an hour of googling and debugging if I'd knew that upfront haha